docs/menu.yml

```diff
@@ -199,6 +199,8 @@ catalog:
                 path: "/en/setup/backend/backend-mysql-monitoring"
               - name: "PostgreSQL Server"
                 path: "/en/setup/backend/backend-postgresql-monitoring"
+              - name: "AWS-DynamoDB Server"
```

Comment:
```suggestion
              - name: "AWS DynamoDB"
```

This should be placed in the AWS monitoring too.

<img width="291" alt="image" src="https://user-images.githubusercontent.com/5441976/218288490-2f54b77b-2894-431e-846b-d5b03e29053a.png">

Also, you should update UI repository to add this feature into two places(database and AWS catalogs).